doc/Metrics.md

```diff
@@ -5,15 +5,17 @@ EdgeHub is intrumented to collect the following metrics:
 | MetricName        | Description           | Unit  |
 | ------------- |:-------------:|:-----:|
 | EdgeHubToCloudMessageLatencyMs | Time taken by EdgeHub to send a message to the cloud | Milliseconds |
-| EdgeHubToCloudMessageSentCount | Number of messages sent by EdgeHub to the cloud | Count per reporting interval* |
+| EdgeHubToCloudMessageSentCount | Number of messages sent by EdgeHub to the cloud | Count resets each reporting interval* |
+| EdgeHubConnectedClientGauge | Number of clients/devices currently connected to EdgeHub | Count |
 | EndpointMessageStoredLatencyMs | Time taken by EdgeHub to acknowledge receipt of a message | Milliseconds |
-| EndpointMessageStoredCount | Number of messages stored by EdgeHub | Count per reporting interval* | 
+| EndpointMessageStoredCount | Total number of messages stored by EdgeHub | Last recorded total | 
+| EndpointMessageDrainedCount | Total number of messages sent to cloud by EdgeHub | Last recorded total |
 | MessageEntityStorePutOrUpdateLatencyMs | Time taken by EdgeHub to record a message in an internal reference counting db store | Milliseconds |
 | SequentialStoreAppendLatencyMs | Time taken by EdgeHub to store a message in an append log | Milliseconds | 
 | DbGetLatencyMs | Time taken by EdgeHub to get a message from the store-and-forward db | Milliseconds | 
 | DbPutLatencyMs | Time taken by EdgeHub to write a message to the store-and-forward db | Milliseconds | 
 
-\* EdgeHub reports metrics to InfluxDb every 20s. Counters are reset after each reporting interval so that if the time series is summed up over a large interval, it returns the true sum as opposed to a sum of sums.
+\* EdgeHub reports metrics to InfluxDb every 5s. Counters are reset after each reporting interval so that if the time series is summed up over a large interval, it returns the true sum as opposed to a sum of sums.
```

Comment:
Isn't _InfluxDb_ just one of many possible backends? Should we just say that the metrics are reported out every 5s?

Reply:
Yes InfluxDb is just one of the many possible backends. So much of this article talks about integration with InfluxDb. We might have to do some rework of this doc when we add another backend.